src/System.Net.Primitives/tests/UnitTests/CookieInternalTest.cs

```diff
@@ -80,33 +80,5 @@ public void CookieParserGetServer_SetCookieHeaderValue_Success(string cookieStri
             int expectedCookieCount = expectedStrings.Length >> 1;
             Assert.Equal(expectedCookieCount, cookieCount);
         }
-
-        [ActiveIssue(22925)]
```

Comment:
Are there any other test methods testing CookieParser.Get()? If not, we should correct test data and keep this test.

Reply:
> The relevant code is already covered by the regular cookie parsing tests.

Oh...I missed this.